src/store/type.ts

```diff
@@ -925,6 +927,11 @@ export type SettingStoreTypes = {
     action(payload: { currentTheme: string }): void;
   };
 
+  SET_EDITOR_FONT: {
+    mutation: { editorFont: "default" | "os" };
+    action(payload: { editorFont: "default" | "os" }): void;
+  };
```

Comment:
`EditorFontType`変え忘れかも？
```suggestion
  SET_EDITOR_FONT: {
    mutation: { editorFont: EditorFontType  };
    action(payload: { editorFont: EditorFontType  }): void;
  };
```